sys/include/net/asymcute.h

```diff
@@ -129,29 +143,29 @@ extern "C" {
  * For default values, see section 7.2 -> Nretry: 3-5.
  */
 #ifndef CONFIG_ASYMCUTE_N_RETRY
-#define CONFIG_ASYMCUTE_N_RETRY            (3U)
+#define CONFIG_ASYMCUTE_N_RETRY         (3U)
 #endif
 /** @} */
 
 #ifndef ASYMCUTE_BUFSIZE
 /**
  * @brief   Default buffer size used for receive and request buffers
  */
-#define ASYMCUTE_BUFSIZE            (1 << CONFIG_ASYMCUTE_BUFSIZE_EXP)
+#define ASYMCUTE_BUFSIZE                (1 << CONFIG_ASYMCUTE_BUFSIZE_EXP)
```

Comment:
Is this buffer really required to be a power of two?

Reply:
I can't find any reason why this should be power of 2 actually...

Reply:
See https://github.com/RIOT-OS/RIOT/pull/15240#discussion_r508656044 and onwards. I believe @akshaim got confused by our somewhat-convention to use powers of two as buffer sizes in general.

Reply:
@leandrolanzieri @miri64  May I squash and rebase as its the first commit I need to rewrite ?

Reply:
Yes of course.

Reply:
Should I deprecate the macro `CONFIG_ASYMCUTE_BUFSIZE_EXP` as its part of the previous release or should I remove it altogether ?

Reply:
I'd say it's ok to deprecate it.

Reply:
Deprecated :+1: